openstudiocore/resources/model/OpenStudio.idd

```diff
@@ -10998,26 +10999,32 @@ OS:HeatPump:WaterToWater:EquationFit:Heating,
        \type object-list
        \required-field
        \object-list ConnectionNames
-   N1,  \field Rated Load Side Flow Rate
+   N1,  \field Reference Load Side Flow Rate
```

Comment:
At first glance I was going to protest that this was going to be an API change. When I looked at the related Class implementation I found that the old name is still used in the API and the implementation maps to the new IDD name in the ModelObject. So I was almost on board. There is still a breaking API change that I will comment on in the header.